docs/guide/writing-tasks.md

```diff
@@ -126,16 +126,19 @@ test_task:
 ```
 
 A `fingerprint_script` is an optional field that can specify a script that will be executed and console output of which
-will be used as a fingerprint for the given task. By default task name is used as a fingerprint value.
+will be used as a fingerprint for the given task. By default the task name is used as a fingerprint value.
```

Comment:
change to "be used as a key for the given cache"

Reply:
fixed